doc/conf.py

```diff
@@ -63,8 +63,8 @@
 
 # General information about the project.
 project = 'PennyLane-Qiskit'
-copyright = "Copyright 2019, Xanadu Quantum Technologies Inc."
-author = 'Carsten Blank, Xanadu Inc.'
+copyright = "Copyright 2022, Xanadu Quantum Technologies Inc."
```

Comment:
To make this consistent:
```suggestion
copyright = "Copyright 2022, Xanadu Quantum Technologies"
```